### PR TITLE
Add test case for very large targets.json

### PR DESCRIPTION
**Pull request by xcheng-here**
Signed-off-by: cheng.xiang <ext-cheng.xiang@here.com>

### CODE REVIEW

File: tuf_vectors/uptane.py [hunk 1]
```diff
@@ -4662,3 +4662,51 @@ def __targets(hardware_id: str, ecu_identifier: str = None) -> list:
...
+            'too_large' : "TRUE",
+        }
+
+        SNAPSHOT_KWARGS = {
+            'snapshot_keys_idx': SNAPSHOT_KEYS_IDX,
+        }
+
+        TIMESTAMP_KWARGS = {
+            'timestamp_keys_idx': TIMESTAMP_KEYS_IDX,
+        }
```
Comment (patrickvacek) [inline tuf_vectors/uptane.py:4695]: I think you're missing an expected error here. As it is, it looks like this test is expected to succeed.

Comment (xcheng-here) [reply tuf_vectors/uptane.py:4699]: Yes, you are right. I modified it.

Comment (xcheng-here) [reply tuf_vectors/uptane.py:4699]: And it's also relate to your another question. Actualizr will fail when it's trying to fetch meta, there will be a error message from curl. And I think we don't throw an exception for curl failure currently. So actually I'm not very happy with the error I'm using here. Maybe the best way would be checking our exception related code ( IIRC there is a jira case for this task??) and add some new exceptions.

Comment (xcheng-here) [reply tuf_vectors/uptane.py:4699]: The curl error message is like this:
curl error 63 (http code 200): Maximum file size exceeded

Comment (patrickvacek) [reply tuf_vectors/uptane.py:4699]: Okay, that error looks okay. The question is then just how to make sure that tuf-test-vectors picks it up correctly. @lbonn can you consult on this? I want to make sure I'm not suggesting something that conflicts with what you're doing.

**Pull request merged**